packages/core/stories/banner/banner.doc.stories.mdx

```diff
@@ -29,28 +28,28 @@ You should always position the Banner component in the main content area, direct
 the navigation or in the related container component.
 
 <Canvas>
-  <Story id="lab-banner--default" />
+  <Story id="core-banner--default" />
 </Canvas>
 
-If you need to share multiple notifications of the same state with your users, you should summarize them in a single Banner
-
-If you have multiple notifications of different states, you should stack the respective banners from top to bottom,
-int the following order: Information, Error, Warning, Success.
-
 ## Banner Status
 
 Banner contains four statuses `info`, `warning`, `error` and `success`. If not specified, the default value is `info`.
 
 <Canvas>
-  <Story id="lab-banner--statuses" />
+  <Story id="core-banner--statuses" />
 </Canvas>
 
+If you need to share multiple notifications of the same state with your users, you should summarize them in a single Banner
+
+If you have multiple notifications of different states, you should stack the respective banners from top to bottom,
+in the following order: Information, Error, Warning, Success.
+
 ## Emphasized
 
 Banner can be emphasized by passing in `emphasize={true}`.
 
```

Comment:
Can we add more detail about what happens to a Banner when it's emphasized?

Reply:
How about this?